src/stories/HeatmapVisScales.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import type { Meta, Story } from '@storybook/react/types-6-0';
+import HeatmapVisStoriesConfig from './HeatmapVis.stories';
+import {
+  HeatmapVis,
+  HeatmapVisProps,
+  ScaleType,
+  getDomain,
+  getMockDataArray,
+} from '../packages/lib';
+
+const dataArray = getMockDataArray('/nD_datasets/twoD');
+const domain = getDomain(dataArray.data);
+const logSafeDomain = getDomain(dataArray.data, ScaleType.Log);
+
+const Template: Story<HeatmapVisProps> = (args) => <HeatmapVis {...args} />;
+
+export const SymLogScale = Template.bind({});
+SymLogScale.storyName = 'Symlog Scale';
+SymLogScale.args = {
+  dataArray,
+  domain,
+  scaleType: ScaleType.SymLog,
+};
+
+export const LogScale = Template.bind({});
+LogScale.args = {
+  dataArray,
+  domain: logSafeDomain,
+  scaleType: ScaleType.Log,
+};
+
+export const SqrtScale = Template.bind({});
+SqrtScale.args = {
+  dataArray,
+  domain: logSafeDomain,
+  scaleType: ScaleType.Sqrt,
+};
```

Comment:
new story for the square root scale

Reply:
I was meaning to do this, thanks :slightly_smiling_face: !

You could use a dedicated `sqrtSafeDomain` instead of the `logSafeDomain` and make the story name clearer:
```SqrtScale.storyName = 'Square root scale';```

Reply:
> You could use a dedicated sqrtSafeDomain instead of the logSafeDomain and make the story name clearer:

🎣 had done it for the line vis scale stories but not here.